### PR TITLE
Bug/1772 errors when submitting files

### LINKED ISSUE #1772: As a TDP user, I see errors when attempting to submit files
## Description
When attempting to upload data files, I receive a error message from the frontend.

## Action Taken
After login, click "data files", search any year/qtr, upload file for section 1, click submit.

## What I expected to see
A successful submission message.

## What I did see
In addition to the "network error" below, I've also seen 500 error.
image
image

## Other Helpful Information

+ URL of the page I was on: /v1/data-files/ across all dev environments
+ Browser and version: Chrome/latest
+ Operating System: Windows
+ Is the issue repeatable?:  yes
+ Has the issue occurred more than once?: yes

### PR DESCRIPTION
**Pull request by andrew-jameson**
## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1772 

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.

1. Changes must be tested on deployed environments -- this functionality worked locally.
1. After logging in to TDP, proceed to upload a file and verify that you receive green success message.
2. Verify via Django Admin that data files are still scanned from CLAMAV

> *Demo GIF(s) and screenshots for testing procedure*
image

## Tabletop

- [ ] Ready for Tabletop (Reviewer check this)
- [x] Tabletop scheduled (Assignee check this)
- [ ] Tabletop approval 

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] Users are able to upload files to deployed sites
+ [x] Users can still download files
+ [x] Files are still scanned from CLAMAV
+ [x] **`lfrohlich`** and/or **`adpennington`**  confirmed that ACs are met.

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [x] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)
  + [x] Backend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [x] Are backend code style checks passing on CircleCI?
+ [x] Are frontend code style checks passing on CircleCI?
+ [x] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [x] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [x] Does this PR provide background for why coding decisions were made?
+ [x] If this PR introduces backend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [ ] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [x] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented? 

### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
+ [ ] the purpose of the research
+ [ ] methods used to conduct the research 
+ [ ] who participated in the research
+ [ ] what was tested and how
+ [ ] impact of research on TDP
+ [ ] (_if applicable_) final design mockups produced for TDP development

### CODE REVIEW

File: .github/workflows/deploy-on-label.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
         echo "Current PR state: $STATE"
     - name: Circle CI Deployment Trigger
       id: curl-circle-ci
-      if: steps.get-pr-state.outputs.STATE == 'success'
+      #if: steps.get-pr-state.outputs.STATE == 'success'
```
Comment (andrew-jameson) [inline .github/workflows/deploy-on-label.yml:63]:
```suggestion
      if: steps.get-pr-state.outputs.STATE == 'success'
```

**Comment by andrew-jameson** [comment]
This can be merged as-is but a nice TODO would be to extend work done via #1774 to ensure AV_SCAN_URL is populated.

**Comment by raftmsohani** [comment]
@andrew-jameson : could you explain what the issue was?

File: tdrs-backend/tdpservice/settings/cloudgov.py [hunk 2]
```diff
@@ -119,6 +119,7 @@ class Development(CloudGov):
 
     # https://docs.djangoproject.com/en/2.0/ref/settings/#allowed-hosts
     ALLOWED_HOSTS = ['.app.cloud.gov']
+    AV_SCAN_URL = os.getenv('AV_SCAN_URL', 'http://tanf-dev-clamav-rest.apps.internal:9000')
```
Comment (riatzukiza) [inline tdrs-backend/tdpservice/settings/cloudgov.py:122]: what is the problem with it being set in common.py?

**Comment by andrew-jameson** [comment]
> @andrew-jameson : could you explain what the issue was?

Sure, the env var `AV_SCAN_URL` was missing or incomplete so I am trying to make sure it default options are included via settings; by incomplete, I noticed URLs in cloud.gov w/o port number.

File: tdrs-backend/tdpservice/settings/cloudgov.py (hunk 2 above)
Comment (riatzukiza) [reply tdrs-backend/tdpservice/settings/cloudgov.py:122]: sounds good, I'll approve this after I run it locally

**Review by raftmsohani — APPROVED** [review]
LGTM

**Review by ADPennington — APPROVED** [review]
LGTM 👍🏾 

qasp before:
1772p1
qasp afte
1772p2
r:

**Pull request merged**